### PR TITLE
Newsletters: add 209 (2022-07-20)

### PR DESCRIPTION
**Pull request by harding**
- [x] Add citations @harding
- [x] Add lede, releases/RCs, topic links @harding 
- [x] Add commit summaries @harding 
- [x] Client/services section @bitschmidty 

Sorry this isn't entirely done yet.  I ran out of time.  I'll try to get the lede, citations, and commit summaries in by the end of Monday.  (Also maybe do some editing.)

### CODE REVIEW

File: _posts/en/newsletters/2022-07-20-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      about it][] on BitcoinTalk; we will only attempt to summarize a
+      few replies we found most notable.
+
+        - *Hard fork not required:* Vjudeu [suggests][] that a soft fork
+          can create new bitcoins by imbuing transaction outputs paying
+          zero satoshis with special meaning.  For example, when a node
+          opting into the fork sees a zero-sat output, it looks at
+          another part of the transaction for the actual value
+          transferred.  This creates two classes of bitcoins, but
+          presumably the soft for would provide a mechanism to convert
```
Comment (Xekyo) [inline _posts/en/newsletters/2022-07-20-newsletter.md:37]:
```suggestion
          presumably the soft fork would provide a mechanism to convert
```

File: _posts/en/newsletters/2022-07-20-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+            be the case that enough value is being paid through
+            transaction fees alone to keep Bitcoin secure.  In short,
+            Bitcoin may already be past the point where it needs its
+            subsidy to obtain sufficient PoW security.  (Though the
+            subsidy also provides other benefits at present, as
+            discussed in the summary below for Bram Cohen's thread.)
+
+            Towns points out that Peter Todd's results depend on a
+            constant average rate of bitcoins being lost each year, but
+            this conflicts with a what Towns thinks should be a
```
Comment (Xekyo) [inline _posts/en/newsletters/2022-07-20-newsletter.md:69]:
```suggestion
            this conflicts with what Towns thinks should be a
```

**Review by Xekyo — COMMENTED** [review]
Thanks for reading all that for all of us.

**Comment by adamjonas** [comment]
Kudos on summarizing the wide-ranging  long-term block reward discussion

**Review by Xekyo — COMMENTED** [review]
Updates look good to me. Sorry, I didn't get around to checking the links, something isn't building right and I don't have time to look into that tonight.

**Comment by harding** [comment]
Pushed all promised updates.  So sorry everything was late this week.  Thanks @Xekyo and @adamjonas for the reviews, especially given the draft status.  Thanks @bitschmidty for his section.  And thank y'all for just generally being awesome every week!  :-D

File: _posts/en/newsletters/2022-07-20-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+          Maxwell [posts][maxwell pi] to BitcoinTalk that there's no reason to
+          believe that paying miners an amount of coins equal to the
+          average rate of lost coins will provide sufficient PoW
+          security, and that there are cases where it could overpay for
+          PoW security.  If a perpetual issuance can't guarantee security
+          and if it has a significant likelihood of producing additional
+          problems, it seems preferable to stick with a finite subsidy
+          that---while it also can't guarantee security---at least it
+          doesn't produce additional problems and is already
+          accepted by all Bitcoiners (implicitly or explicitly).
```
Comment (glozow) [inline _posts/en/newsletters/2022-07-20-newsletter.md:57]: apologies, this sentence might be 100% grammatically correct but confused me...?
```suggestion
          PoW security.  If a perpetual issuance can't guarantee security
          and if it has a significant likelihood of producing additional
          problems, it seems preferable to stick with a finite subsidy
          that---while also can't guarantee security---at least
          doesn't produce additional problems and is already
          accepted by all Bitcoiners (implicitly or explicitly).
```

**Review by glozow — COMMENTED** [review]
LGTM. And +1, impressive "Long-term block reward ongoing discussion" summary

**Comment by bitschmidty** [comment]
Reviewed @harding  updates and links, took @Xekyo suggestion on the client services section, made a small change to the BDK summary, squashed.

Thanks to @harding @Xekyo @glozow @adamjonas for contributing this week!

**Pull request merged**